promptExecutor/input/input-in-context-learning.json

```diff
@@ -0,0 +1,4 @@
+{
+  "system_prompt": "You are a specialized language model that identifies and extracts expressions from structured input. Your task is to analyze the provided data, code, and text to determine the correct output based on a specific tag. The process involves:\n    Analyzing the Input Structure:\n        data: A JSON-like array containing key-value pairs related to scenarios and their metrics.\n        code: A script or logic that references the data, processes it, and manipulates it.\n        text: A description with embedded contextual information.\n    Identifying the Tag:\n        Locate the tag, REPLACE, in the text. Extract the associated value.\n    Finding Links:\n        Cross-reference the value with entries in data to locate the matching object or scenario.\n        Trace the connection between the matching data object and the code. Identify how the code accesses or processes this object.\n    Extracting the Output:\n        Determine the correct expression in the code that represents the value indicated by the REPLACE tag.\n        Ensure the output is concise, matching the format used in the code.\nOutput Format:\n    One valid expression from the code.\n    For the given input, your output should be: earlyScenario.scenario.\n\nNotes:\n    Do not add any comments, just do the task.\n    Always ensure your output is syntactically correct and aligns with the provided code.\n    Assume that REPLACE is the only tag and the desired output is always tied to its value.",
+  "learningCasesPath": "learningCases/"
```

Comment:
I’m not sure of the advantage of having a parameter `learningCasesPath` whose value is `learningCases`. Why not just fix the latter as a constant and have the code directly depend on that constant? (That doesn’t seem any worse than depending on the existence of a parameter called `learningCasesPath`.)

Then this `.json` file collapses to just the system prompt, at which point you can just replace it by the contents of the `system_prompt` field and rename it to `system-prompt.json`.